sdks/python/apache_beam/io/textio.py

```diff
@@ -86,7 +90,19 @@ def __init__(self,
                coder,
                buffer_size=DEFAULT_READ_BUFFER_SIZE,
                validate=True,
-               skip_header_lines=0):
+               skip_header_lines=0,
+               header_processor_fns=(None, None)):
+    """Initialize a _TextSource
+
+    Args:
+      header_processor_fns (tuple): a tuple of a header_matcher function
+        and a header_processor function. The header_matcher should return True
+        for all lines at the start of the file that are part of the file header
+        and False otherwise. These header_lines will be skipped over when
```

Comment:
<!--new_thread; commit:63511ddb8caf9e5c12888ef8a4d1071e0ea6c168; resolved:0-->
Add a comment: "Please refer to documentation in class \`ReadFromText\` for rest of the arguments."

Reply:
Done.